core/packet-tx.go

```diff
@@ -49,6 +57,10 @@ func SendTransferMsg(src, dst *ProvableChain, amount sdk.Coin, dstAddr fmt.Strin
 	}
 
 	if txs.Send(src, dst); !txs.Success() {
+		channelLogger.Error(
+			"failed to send transfer message",
+			errors.New("failed to send transfer message"),
+		)
 		return fmt.Errorf("failed to send transfer message")
```

Comment:
Why do you create two different error instances?
You should create one and use it in both the log and the function result.

Reply:
Remove errors.New